ios/RCTGoogleAnalyticsBridge/RCTGoogleAnalyticsBridge/RCTGoogleAnalyticsBridge.m

```diff
@@ -138,6 +138,14 @@ - (NSDictionary *)constantsToExport
                                                         target:targetUrl] build]];
 }
 
+RCT_EXPORT_METHOD(setCustomDimensionValue:(NSString *)trackerId dimensionIndex:(NSInteger)dimensionIndex dimensionValue:(NSString *)dimensionValue)
+{
+    id<GAITracker> tracker = [[GAI sharedInstance] trackerWithTrackingId:trackerId];
+    
+    [tracker set:[GAIFields customDimensionForIndex:dimensionIndex] value:dimensionValue];
```

Comment:
No apparently it is a method on the `HitBuilder` type.
From the docs:
```java
// Set screen name:
t.setScreenName("Home Screen");

// Send the custom dimension value along with the screen name:
t.send(new HitBuilders.ScreenViewBuilder()
    .setCustomDimension(1, "premiumUser")
    .build()
);
```

One way would be to cache the custom dimension value, along with a boolean to say if it has been sent or not (since it only needs to be sent once). That would give it the same behavior as on iOS.

But that requires a lot more moving parts, and checking that cache in the screen/event tracking-methods. It might be just as easy to go with my previous idea of sending in a screenName (or event) along with the custom dimension value. This would then give parity between the iOS and Android implementation.